### PR TITLE
218221 case - changes in ILS tests

### PR DESCRIPTION
**Pull request by kkrampa**
@gcapalbo 
http://manage.dimagi.com/default.asp?218221#1102597
commit by commit

### CODE REVIEW

File: custom/ilsgateway/tanzania/handlers/generic_stock_report_handler.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 from django.conf import settings
 from corehq.apps.commtrack.exceptions import NotAUserClassError
-from corehq.apps.commtrack.sms import process
+from corehq.apps.commtrack.sms import process, SMSError
```
Comment (dimagimon) [inline custom/ilsgateway/tanzania/handlers/generic_stock_report_handler.py]: F401 'SMSError' imported but unused

File: custom/ilsgateway/tanzania/handlers/soh.py [hunk 2]
```diff
@@ -66,30 +67,29 @@ class SOHHandler(GenericStockReportHandler):
...
-                'product_list': ' '.join(
-                    sorted([SQLProduct.objects.get(product_id=product_id).code for product_id in diff])
-                )
+        with localize(self.user.get_language_code()):
+            reported_earlier = StockState.objects.filter(
+                case_id=self.sql_location.couch_location.linked_supply_point().get_id,
+                last_modified_date__gte=datetime.utcnow() - timedelta(days=7)
+            ).values_list('product_id', flat=True)
+            expected_products = set(
+                self.location_products.exclude(product_id__in=reported_earlier).values_list('product_id', flat=True)
```
Comment (dimagimon) [inline custom/ilsgateway/tanzania/handlers/soh.py]: E501 line too long (116 > 115 characters)

File: custom/ilsgateway/tests/handlers/messageinitatior.py [hunk 3]
```diff
@@ -1,12 +1,29 @@
+from django.utils import translation
```
Comment (dimagimon) [inline custom/ilsgateway/tests/handlers/messageinitatior.py]: F401 'translation' imported but unused

File: custom/ilsgateway/tests/handlers/stockonhand.py [hunk 4]
```diff
@@ -1,8 +1,11 @@
+from django.utils import translation
```
Comment (dimagimon) [inline custom/ilsgateway/tests/handlers/stockonhand.py]: F401 'translation' imported but unused

**Comment by kkrampa** [comment]
@gcapalbo  Updated (last 4 commits)

**Pull request merged**